tests/test_config_files/import_tiff/import_tiff_resized_scale_factors.json

```diff
@@ -1,7 +1,6 @@
 {
   "pipeline": "eogrow.pipelines.import_tiff.ImportTiffPipeline",
   "**global_config": "${config_path}/../global_config.json",
-  "tiff_folder_key": "input_data",
```

Comment:
don't remove it everywhere. We want tests to detect a validation problem if someone accidentally deletes the key (yeah yeah i know it would probably crash elsewhere).

Reply:
done. removed it in half the cases.